wp-content/plugins/goonj-blocks/src/render.php

```diff
@@ -83,13 +82,22 @@
   $address = $action_target[$target_info['address']];
   $contribution_link = $target_info['contribution_link'];
   $address_label = $target_info['address_label'];
+  $volunteer_name = $action_target[$target_info['volunteer_name']];
 
   ?>
     <div class="wp-block-gb-heading-wrapper">
         <h2 class="wp-block-gb-heading"><?php echo esc_html($heading_text); ?></h2>
     </div>
     <table class="wp-block-gb-table">
         <tbody>
+            <?php if ($target === 'dropping-center') : ?>
+            <tr class="wp-block-gb-table-row">
+                <td class="wp-block-gb-table-cell wp-block-gb-table-header">Volunteer name</td>
+                <td class="wp-block-gb-table-cell"><?php echo esc_html($volunteer_name); ?></td>
+            </tr>
```

Comment:
Indent this block

Reply:
@pokhiii, Indentation is already fixed here.